src/components/organisms/AuthenticationModal/LoginForm/LoginForm.tsx

```diff
@@ -94,6 +103,25 @@ const LoginForm: React.FunctionComponent<PropsType> = ({
 
       <h2>Log in</h2>
 
+      {hasAlternativeLogins && (
+        <>
+          <p>Quick log in with</p>
+
+          <div className="LoginForm__alternative-logins">
+            {hasSSOProvider && (
+              <img
+                className="LoginForm__sso-login"
+                src={SSO_LOGIN_ICON}
+                onClick={signInSSO}
+                alt="SSO login"
+              />
+            )}
```

Comment:
Instead of just showing an icon here, it might be better for us to show some text as well?

Reply:
Have you looked at https://firebase.google.com/docs/reference/js/firebase.auth.Auth?authuser=2#fetchsigninmethodsforemail at all? Not sure if it would be relevant here.. but figured I would mention it in case.

Reply:
Not sure what it's about 

> Have you looked at https://firebase.google.com/docs/reference/js/firebase.auth.Auth?authuser=2#fetchsigninmethodsforemail at all? Not sure if it would be relevant here.. but figured I would mention it in case.

Reply:
@mike-lvov we got some notes on sign-in flow / descriptive text requests based on what we shared with client yesterday, I'll tag you in main issue if we need to add something here

Reply:
Also adding @malaikaaiyar to support some textual additions / revisions to this flow